debian/control

```diff
@@ -9,7 +9,8 @@ Build-Depends: check,
                libboost-dev (>= 1.41),
                libboost-program-options-dev (>= 1.41),
                libssl-dev,
-               scons (>= 2)
+               scons (>= 2),
+               libasio-dev
```

Comment:
Putting libasio-dev above scons would have made a patch a tad smaller ;) Also, if this is made a build dependency on debian, whoud not it make sense to specify a minimum version as well? I know, we check for it during build anyway, but it is further down the line.